lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -183,8 +183,8 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   }
   foreach ($address as $ns_key => $drupal_key) {
     $field = $user->field_address[LANGUAGE_NONE][0];
-    if (isset($field[$drupal_key]['value'])) {
-      $ns_user[$ns_key] = $field[$drupal_key]['value'];
+    if (isset($field[$drupal_key])) {
+      $ns_user[$ns_key] = $field[$drupal_key];
```

Comment:
The reason we use the `'value'` index of the array is for Drupal fields, which expose a plain-text value but then also a `'safe_value'` index that has an escaped version of that string. From checking on my local, it seems like that isn't the case with these address fields so I _think_ your change should be good.

The remaining question is if there are some cases where a field on this `$address` array could have the `value` and `safe_value` indexes rather than just a plain string value. Do you know if this function is ever called outside of the new user registration flow?